src/briefcase/platforms/windows/__init__.py

```diff
@@ -124,14 +127,145 @@ def default_packaging_format(self):
     def verify_tools(self):
         super().verify_tools()
         WiX.verify(self.tools)
+        if self._windows_sdk_needed:
+            WindowsSDK.verify(self.tools)
 
-    def package_app(self, app: BaseConfig, **kwargs):
+    def add_options(self, parser):
+        super().add_options(parser)
+        parser.add_argument(
+            "--file-digest",
+            help="File digest algorithm to use for creating file signatures; defaults to sha256.",
+            default="sha256",
+            required=False,
+        )
+        parser.add_argument(
+            "--cert-store-location",
+            help="Location of stores containing the certificate; defaults to Current User.",
+            choices=["Current User", "Local Machine"],
+            default="Current User",
+            required=False,
+        )
+        parser.add_argument(
+            "--cert-store",
+            help="Store containing the certificate; defaults to the Personal store.",
+            default="My",
+            required=False,
+        )
+        parser.add_argument(
+            "--timestamp-url",
+            help=(
+                "URL for the Timestamp Authority server to timestamp "
+                "the code signing; defaults to timestamp.digicert.com."
+            ),
+            default="http://timestamp.digicert.com",
```

Comment:
I would not say that digicert's presence here should be considered a recommendation per se....however,  of the well known entities available for providing such a timestamp, they are one of the few that have proven themselves trustworthy to their users.

Insofar as choosing any default here, I'd expect the criteria to be inherently subjective; there isn't a de facto choice.

So, I'm open to a different default....or not using a default at all. However, we should not let users sign without one; the consequences are entirely unobvious but devastating in the long term. So, without a default, we should error and require one to be specified.

Reply:
Given the sensitivity of the issue, I think a prefer a historically reliable sensible default over letting the user select to avoid commercial endorsement.